cocos/ui/editbox/edit-box-impl.ts

```diff
@@ -279,41 +279,46 @@ export class EditBoxImpl extends EditBoxImplBase {
         const transform = node._uiProps.uiTransformComp;
         if (transform) {
             Vec3.set(_vec3, -transform.anchorX * transform.width, -transform.anchorY * transform.height, _vec3.z);
+            Mat4.transform(_matrix, _matrix, _vec3);
         }
 
-        Mat4.transform(_matrix, _matrix, _vec3);
-
         if (!node._uiProps.uiTransformComp) {
             return;
         }
 
         const camera = director.root!.batcher2D.getFirstRenderCamera(node);
         if (!camera) return;
 
-        // camera.getWorldToCameraMatrix(_matrix_temp);
         camera.node.getWorldRT(_matrix_temp);
         const m12 = _matrix_temp.m12;
         const m13 = _matrix_temp.m13;
         const center = visibleRect.center;
         _matrix_temp.m12 = center.x - (_matrix_temp.m00 * m12 + _matrix_temp.m04 * m13);
         _matrix_temp.m13 = center.y - (_matrix_temp.m01 * m12 + _matrix_temp.m05 * m13);
 
-        Mat4.multiply(_matrix_temp, _matrix_temp, _matrix);
```

Comment:
shouldn't multiply `_matrix` (node's matrix) to `_matrix_temp` and then do scale separately later.